build_tools/cmake/FindBazel.cmake

```diff
@@ -0,0 +1,48 @@
+#[=======================================================================[.rst:
```

Comment:
For this file (and all other `*.cmake` files), is it your own code? If so it needs the CompilerGym license header. If it's taken from another project, does it need a license header + attribution for that project?

Reply:
Some parts of the code come from IREE's Bazel to CMake autoconversion tool. That is under Apache 2.0 license and it needs attribution.

Reply:
Okay, in that case I think that the *.cmake files should have IREE's license header added